genie-core/src/main/java/com/netflix/genie/core/util/MetricsConstants.java

```diff
@@ -102,6 +102,26 @@ private TagKeys() {
          */
         public static final String FAILURE = "failure";
 
+
+        /**
+         * Constant representing a no id found for tags added to metrics.
+         * E.g this values is used for a missing cluster id when tagging
+         * metrics by cluster id
+         */
+        public static final String NO_ID_FOUND = "<no id>";
+
+        /**
+         * Constant representing no cluster found when tagging metrics
+         * with cluster tags.
+         */
+        public static final String NO_CLUSTER_FOUND = "NO_CLUSTER_FOUND";
```

Comment:
Nitpick: "No cluster found" makes it sounds like something went wrong.
In reality either we have a cluster or we don't, and in case we don't it's expected, so `"NO CLUSTER"` would be a little more readable by someone looking at metrics that is not familiar with the details. 
Also no need to underscore the literal.